README.md

```diff
@@ -82,3 +82,7 @@ From our experience, this will not happen if you use the virtual environements o
   - session -- Different abstract classes for wrappers.
 - examples -- examples of usage.
 - tests -- unittesting of the code
+
+## Acknowledgements
+
+Special thanks to [Enthought, Inc](https://www.enthought.com/); [Centre Internacional de Mètodes Numèrics a l'Enginyeria (CIMNE)](https://cimne.com/) and the [University of Jyväskylä](https://www.jyu.fi/en) for collaborating with [Fraunhofer IWM](https://www.iwm.fraunhofer.de/en.html) to devise some of the ideas behind OSP-core.
```

Comment:
Suggestion:

The variant of the SimPhoNy Python package originates from the European Project SimPhoNy (Project Nr. 604005, website: www.simphony-project.eu).
We would like to thank our partners and acknowledge their important contributions to the first implementation, that is available here: https://github.com/simphony/simphony-common.
Native implementation of the core cuds object and the class generation by the SimPhoNy team at Fraunhofer IWM. Special thanks to [Enthought, Inc](https://www.enthought.com/); [Centre Internacional de Mètodes Numèrics a l'Enginyeria (CIMNE)](https://cimne.com/) and the [University of Jyväskylä](https://www.jyu.fi/en) for collaborating with [Fraunhofer IWM](https://www.iwm.fraunhofer.de/en.html) to devise some of the ideas behind OSP-core.

Reply:
New suggestion:

The OSP-core Python package originates from the European Project SimPhoNy (Project Nr. 604005, website: www.simphony-project.eu). We would like to acknowledge and thank our project partners, especially [Enthought, Inc](https://www.enthought.com/), [Centre Internacional de Mètodes Numèrics a l'Enginyeria (CIMNE)](https://cimne.com/) and the [University of Jyväskylä](https://www.jyu.fi/en), for their important contributions to some of the core concepts of OSP-core, which originally demonstrated under the project https://github.com/simphony/simphony-common.

Reply:
@kysrpex what do you think?

Reply:
> @kysrpex what do you think?

I like it better than the previous suggestion.